pages/glossary/user-agent-controlled.md

```diff
@@ -0,0 +1,31 @@
+---
+title: User-Agent Controlled Component
+key: user-agent-controlled-component
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+A _User-Agent Controlled Component_ is an [HTML element][namespace element] for which all the following are true:
+
+- the element has an [implicit role][] which is a [semantic `widget`][semantic role]; and
+- the [computed values][] of the element's `height` and `width` CSS properties do not depend on content provided by the author; and
+- the [computed values][] of the element's `height` and `width` CSS properties do not depend on any CSS property with a [cascaded value][] with "Author" [origin][].
```

Comment:
This feels off. For starts, font-size is a CSS property often set by the author that changes the width and height.

I'm also not sure whether this then includes things that feel like they shouldn't. An input[type=text] element can be adjusted by the size attribute. I don't think that then creates an author-origin CSS property.